src/main/cpp/subsystems/ScoringSubsystem.cpp

```diff
@@ -58,10 +59,19 @@ void ScoringSubsystem::SpinSpeaker() {
     m_speakerUpperSpinnyBoi.Set(ScoringConstants::kSpeakerUpperSpeed);
 }
 
+void ScoringSubsystem::SpinSubwoofer() {
```

Comment:
The upper-level `SpinVectorSide`, `StartScoringRamp`, `GetMotorAtScoringSpeed`, and `GetMotorFreeWheel` were never updated to reflect the addition of `ScoringDirection::Subwoofer`